### PR TITLE
payload: reuse supported_arches

### PR DESCRIPTION
**Pull request by beraldoleal**
supported_arches variable is defined but not used during manifest create step. With this patch we can easly disable arches for quick development test.

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]
This is nice.

cc @portersrc

**Review by ldoktor — APPROVED** [review]
lol, this is exactly what I was thinking about while reviewing PR that touched this file. The only cons is that the `purge_previous_manifests` clears all the manifests and not just the ones that will be re-created. As a result running this with both (x86_64 and s390x) and then re-running it with let's say only s390x the x86_64 manifest would be gone as well. Anyway I don't think it's a big deal... (but it adds to the question whether we need the `purge_previous_manifests` at all)

File: install/pre-install-payload/payload.sh [hunk 1]
```diff
@@ -57,6 +57,7 @@ function build_payload() {
 
 	tag=$(git rev-parse HEAD)
 
+  manifest_args=()
```
Comment (portersrc) [inline install/pre-install-payload/payload.sh:60]: nit: mind the gap

**Review by portersrc — APPROVED** [review]
LGTM

File: install/pre-install-payload/payload.sh (hunk 1 above)
Comment (ldoktor) [reply install/pre-install-payload/payload.sh:60]: ah, didn't noticed this, @beraldoleal could you please address it, then it should be ready to be merged.

**Comment by portersrc** [comment]
> The only cons is that the purge_previous_manifests clears all the manifests  ... Anyway I don't think it's a big deal... (but it adds to the question whether we need the purge_previous_manifests at all)

purge-previous-manifests looks odd to me on multiple levels. I would handle it in a different issue and cc me on it if it's irking you, @ldoktor 👍

**Review by ldoktor — APPROVED** [review]
Thanks, lgtm now. Let me re-run the hanged job and merge it...

**Pull request merged**